res/values/strings.xml

```diff
@@ -785,6 +815,10 @@
     <string name="menu.view.developer.open.log.folder">Open the log folder</string>
     <string name="menu.view.developer.open.current.log.file">Open current logfile</string>
     <string name="user_location_permission_explanation">Delta Chat needs the location permission in order to show and share your location.</string>
+    <string name="ask_delete_message_desktop">Are you sure you want to delete this message?</string>
```

Comment:
while this string is introduced on desktop, it is not desktop specific any may be used somewhere else as well. therefore, remove the `_desktop` suffix and maybe move it up below `ask_delete_messages` (maybe even that string can be used)

Reply:
considered using `ask_delete_messages` but it says something too different. We'll probably use it when we have multi select messages in desktop.